packages/storybook/layouts.stories.js

```diff
@@ -113,54 +113,88 @@ storiesOf('Layouts', module)
-                    <span>Manage</span>
-                  </a>
-                </li>
-              </ul>
-            </div>
-            <div class="bottom-nav">
-              <ul class="nav-items">
-                <li>
-                  <a href="/">
-                    <iron-icon class="size-l" icon="lumo:user"></iron-icon>
-                    <span>Profile</span>
-                  </a>
-                </li>
-              </ul>
+      <div class="wrapper">
+        <div id="left-nav">
+          <a href="/">
+            <div class="logo">
+              <span class="cxl">C<span>XL</span></span>
             </div>
+          </a>
+          <div class="top-nav">
+            <ul class="nav-items">
+              <li>
+                <a href="trainings" class="active">
+                  <iron-icon class="size-l" icon="cxl:nav-trainings"></iron-icon>
+                  <iron-icon class="size-l" icon="cxl:nav-trainings-active"></iron-icon>
+                  <span>Trainings</span>
+                </a>
+              </li>
+              <li>
+                <a href="roadmap">
+                  <iron-icon class="size-l" icon="cxl:nav-roadmap"></iron-icon>
+                  <iron-icon class="size-l" icon="cxl:nav-roadmap-active"></iron-icon>
+                  <span>Discover</span>
+                </a>
+              </li>
+              <li>
+                <a href="discover">
+                  <iron-icon class="size-l" icon="cxl:nav-discover"></iron-icon>
+                  <iron-icon class="size-l" icon="cxl:nav-discover-active"></iron-icon>
+                  <span>Manage</span>
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="bottom-nav">
+            <ul class="nav-items">
+              <li>
+                <a href="/">
+                  <iron-icon class="size-l" icon="lumo:user"></iron-icon>
+                  <span>Profile</span>
+                </a>
+              </li>
+            </ul>
           </div>
-          <cxl-app-layout> </cxl-app-layout>
         </div>
-      </cxl-layout-institute>
+        <cxl-institute-layout>
+          <h3 slot="sidebar-header">Training Manager</h3>
+          <div class="sidebar-content" slot="sidebar-content">
+            <strong class="extra-heading">Digital analytics minidegree program</strong>
+            <vaadin-details theme="sidebar-track" opened>
+              <div slot="summary">Google Analytics</div>
+              <div>
+                <cxl-institute-sidebar-items>
+                  <cxl-institute-sidebar-item item-type="track" track-value="1:42">
+                    <!-- href class active -->
+                    <a href="trainings/analytics-fundamentals/484381">Analytics fundamentals</a>
+                  </cxl-institute-sidebar-item>
+                  <cxl-institute-sidebar-item item-type="track" track-value="3:14">
```

Comment:
I'm not sure at all what the value of these highly specific components are.

Maybe simpler to use native elements? What's stopping it? `ul > li > span + time` type arrangement

https://stackoverflow.com/questions/31559103/html-time-tag-for-duration

Reply:
It's really hard to understand what you want. At some point, you are saying why is there global css and you got excited about aybolit zero global css/js. We have to make up our mind about what is global and is component. Probably you have to decide.

Reply:
Right amount of custom.

As little as possible, as much as needed.

Where exactly the line is === challenge.